README.md

```diff
@@ -126,6 +134,15 @@ just test-foundry
 
 There are several end-to-end tests in the `e2e/interchaintestv8` directory. These tests are written in Go and use the [`interchaintest`](https://github.com/strangelove-ventures/interchaintest) library. It spins up a local Ethereum and a Tendermint network and runs the tests found in [`e2e/interchaintestv8/sp1_ics07_test.go`](e2e/interchaintestv8/sp1_ics07_test.go). Some of the tests use the prover network to generate the proofs, so you need to provide your SP1 network private key to `.env` for these tests to pass.
 
+> If you are running on a mac with an M chip, you will need to do the following:
+> - Set up Rosetta
+> - Enable Rosetta for Docker (in Docker Desktop: Settings -> General -> enable "Use Rosetta for x86_64/amd64 emulation on Apple Silicon")
+> - Pull the foundry image with the following command:
+> 
+>     ```sh
+>     docker pull --platform=linux/amd64 ghcr.io/foundry-rs/foundry:latest
+>     ```
```

Comment:
Shouldn't interchaintest automatically pull it for you?

Reply:
For me it pulled the arm version which doesn't work. 🤷